content/en/blog/_posts/2024-04-15-devsecops-in-action-k8s-secrets.md

```diff
@@ -0,0 +1,554 @@
+---
+layout: blog
+title: "DevSecOps in Action: Kubernetes Secrets"
+slug: devsecops-in-action-k8s-secrets
+date: 2024-04-15
```

Comment:
How about we bump this to the 9th of May? (change it here and in the filename)

Reply:
> How about we bump this to the 9th of May? (change it here and in the filename)

Works perfectly for me!